modules/jupyter-widget/src/utils.js

```diff
@@ -89,24 +89,39 @@ function getTooltip(pickedInfo) {
 }
 
 function tabularize(json) {
+  // Turns a JSON object of picked info into HTML for a tooltip
   const dataTable = document.createElement('div');
   dataTable.className = 'dataTable';
 
+  // Creates rows of two columns for the tooltip
   for (const key in json) {
+    if (['undefined', 'position', 'index'].includes(key)) {
+      continue; // eslint-disable-line
+    }
     const row = document.createElement('div');
     const header = document.createElement('div');
     header.className = 'header';
     header.innerText = key;
     Object.assign(header.style, {
-      fontWeight: 500,
-      marginRight: '10px'
+      fontWeight: 700,
+      marginRight: '10px',
+      flex: 1
     });
     const value = document.createElement('div');
     value.className = 'value';
-    value.innerText = JSON.stringify(json[key]);
+
+    if (Array.isArray(json[key])) {
+      value.innerText = JSON.stringify(`Array<${json[key].length}>`);
+    } else {
+      value.innerText = JSON.stringify(json[key]);
```

Comment:
- This should be clamped in case the object is very large.
- `JSON.stringify` may [throw exceptions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#Exceptions)